include/envoy/tracing/http_tracer.h

```diff
@@ -6,6 +6,19 @@
 
 namespace Tracing {
 
+/**
+ * Transport tracing context.
+ * It's used to set proper parent/child span relationship on Envoy calls, e.g., ratelimit call.
+ */
+struct TransportContext {
+  std::string request_id_;
+  std::string span_context_;
+
+  bool operator==(const Tracing::TransportContext& rhs) const {
```

Comment:
since this is only needed in test code, put the definition of operator== in a test header.